src/stat.js

```diff
@@ -127,3 +127,111 @@ export function varianceAll(matrix, unbiased, mean) {
     return (sum2 - (sum1 * sum1) / size) / size;
   }
 }
+
+
+/**
+ * Calculate standard deviation column by column of a matrix
+ * @param {Matrix} matrix - Column index
+ * @return {Array}
+ */
+export function stdDeviationByColumn(matrix) {
```

Comment:
this is already implemented as [`matrix.standardDeviation('column', { unbiased: true })`](https://github.com/mljs/matrix/blob/d9119525ff1bf6c108dca89e696598db3d5e0958/src/matrix.js#L1681-L1696)